qiskit_experiments/framework/composite/batch_experiment.py

```diff
@@ -60,7 +62,22 @@ def __init__(self, experiments: List[BaseExperiment], backend: Optional[Backend]
         super().__init__(experiments, qubits, backend=backend)
 
     def circuits(self):
+        circs = []
+        for circ_group in self._batch_circuits():
+            circs.extend(circ_group[0])
```

Comment:
Is this for the backward compatibility?

Reply:
A user may call `circuits`, with the expectation to retrieve the circuits before transpilation